WORKSPACE

```diff
@@ -0,0 +1,16 @@
+new_http_archive(
```

Comment:
This downloads the gmock archive so that we can use it for the unit tests. This is similar to CMake's [ExternalProject](https://cmake.org/cmake/help/v3.0/module/ExternalProject.html) mechanism but friendlier. :)

Here's the documentation for [`new_http_archive`](http://bazel.io/docs/be/workspace.html#new_http_archive).